.github/workflows/ci.yml

```diff
@@ -358,6 +380,7 @@ jobs:
 
   minrust:
     name: minrust
+    needs: basics
     runs-on: ubuntu-latest
```

Comment:
I think it would make sense to include this one in the basic checks.

Reply:
For sure, it's a 30s action. Will add it.